pymongo/bulk.py

```diff
@@ -254,25 +254,38 @@ def gen_unordered(self):
                 yield run
 
     def _execute_command(self, generator, write_concern, session,
-                         sock_info, op_id, retryable, full_result):
+                         sock_info, op_id, retryable, full_result, final_write_concern=None):
         db_name = self.collection.database.name
         client = self.collection.database.client
         listeners = client._event_listeners
 
         if not self.current_run:
             self.current_run = next(generator)
+            self.next_run = None
```

Comment:
next_run should be defined in `__init__`.

Reply:
Done